app/Http/Controllers/PetController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class PetController extends Controller
+{
+    public function show(){
+        return view ('pets',['pets' => self::getPet(),]);
+    }
+    public static function getpet(){
```

Comment:
this can be private scope since it's only called within the class definition. also if returning an array of pets I'd name it `getPets`